source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -0,0 +1,76 @@
+.. _telemetry-about:
+
+Telemetrics
+###########
+
+Telemetry is one of the key features of |CLOSIA|. The |CL| team is focused on
+the quality of the telemetry data, and is moving from issues that are detected
+and reported by users to issues detected and repaired by providers. Ideally,
+these issues will be undetected by users.
+
+On the |CL| team, we refer to *telemetrics*, which is a combination word made from:
```

Comment:
I suggest revising to simply start: "*telemetrics* is a combination word..."

Reply:
@MCamp859 
Pending release of this PR, I take on the AR to update with a link to your new document, the associated feature page here: https://clearlinux.org/features/telemetry
@mvincerx